Concepts/concept_source.md

```diff
@@ -0,0 +1,106 @@
+# Concept Source Type
+
+## Overview
+
+* A concept maps with the concept source type in openMRS. 
```

Comment:
Concepts are often managed within a dictionary (as a collection of concepts). While OpenMRS has it's own dictionary of concepts, other dictionaries may exist in other systems or as standardized reference terminologies (like [LOINC](https://loinc.org) or [ICD](https://en.wikipedia.org/wiki/International_Statistical_Classification_of_Diseases_and_Related_Health_Problems)). The authorities who manage these other concept dictionaries represent "Concept Sources".